web/scripts/app.js

```diff
@@ -1001,15 +1001,17 @@ export class ComfyApp {
 	/**
 	 * Loads all extensions from the API into the window
 	 */
-	async #loadExtensions() {
+    async #loadExtensions() {
```

Comment:
New indent doesn't match the rest of the file.

Reply:
fixed